doc/Makefile

```diff
@@ -23,11 +23,12 @@ help:
 DOCUMENTER_OPTIONS := linkcheck=$(linkcheck) doctest=$(doctest) buildroot=$(call cygpath_w,$(BUILDROOT)) \
     texplatform=$(texplatform)
 
-UnicodeData.txt:
-	$(JLDOWNLOAD) http://www.unicode.org/Public/9.0.0/ucd/UnicodeData.txt
+$(SRCCACHE)/UnicodeData.txt:
+	$(JLDOWNLOAD) "$@" http://www.unicode.org/Public/9.0.0/ucd/UnicodeData.txt
+	$(JLCHECKSUM) "$@"
 
-deps: UnicodeData.txt
-	$(JLCHECKSUM) UnicodeData.txt
+deps: $(SRCCACHE)/UnicodeData.txt
+	cp "$<" UnicodeData.txt
```

Comment:
`JLCHECKSUM` call needs to be here before the use, it won't do anything to have it after the build of the target

Reply:
I was expecting to catch mis-downloads, but I guess we can check for bitrot on the storage device as well.  Checksumming is fast, after all.  :P

Reply:
Right, but to do that earlier, you'd have to make sure it didn't save the file until after the checksum succeeded. And so it's been easier just to do that on first use.